src/libraries/Microsoft.Extensions.Options/gen/Parser.cs

```diff
@@ -467,7 +470,81 @@ private List<ValidatedMember> GetMembersToValidate(ITypeSymbol modelType, bool s
                         continue;
                     }
 
-                    var validationAttr = new ValidationAttributeInfo(attributeType.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat));
+                    string attributeFullQualifiedName = attributeType.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat);
+                    if (SymbolEqualityComparer.Default.Equals(attributeType, _symbolHolder.MaxLengthAttributeSymbol))
+                    {
+                        if (!CacheAttributesData(memberType, _symbolHolder.MaxLengthAttributeSymbol.Name, location, attributeType, ref attributeFullQualifiedName))
+                        {
+                            continue;
+                        }
+                    }
+                    else if (SymbolEqualityComparer.Default.Equals(attributeType, _symbolHolder.MinLengthAttributeSymbol))
+                    {
+                        if (!CacheAttributesData(memberType, _symbolHolder.MinLengthAttributeSymbol.Name, location, attributeType, ref attributeFullQualifiedName))
+                        {
+                            continue;
+                        }
+                    }
+                    else if (_symbolHolder.MinLengthAttributeSymbol is not null &&
```

Comment:
Only this symbol has the `_symbolHolder... is not null` check. Is that intentional?

Reply:
`LengthAttribute` is introduced in 8.0 so if we are generating code on down-level, its symbol can be null. This is why I am making it optional existence in the symbol holder. But I had typo you caught as I was checking MinLengthAttributeSymbol instead. most likely I did that during some refactoring I was doing before submitting the PR. Thanks for the catch!